SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -283,8 +283,21 @@ void USpatialActorChannel::RetireEntityIfAuthoritative()
 	}
 }
 
+void USpatialActorChannel::ValidateChannelNotBroken()
+{
+	// In native Unreal, channels can be broken in certain circumstances (e.g. when unloading streaming levels or failing to process a
+	// bunch). This shouldn't happen in Spatial and would likely lead to unexpected behavior.
+	if (Broken)
```

Comment:
is this a boolean from the base `ActorChannel` class? does unreal have its own checks around this?

Reply:
In native, channels can get broken during processing bunches, but we skip that completely since we don't actually send or receive data through the channels. With the notify functions in the NetDriver overridden, there shouldn't be a way channels get broken in Spatial now, so this log is essentially to flag when that isn't true so we can address it (e.g. if stuff changes on next engine upgrade)